### PR TITLE
Modified pets seeds to utlize faker and cleaned up user emails 

### PR DESCRIPTION
**Pull request by jadekstewart3**
### Describe your change in plain English.
Added faker gem and utilized it in the creation of seed pets, and cleaned up seed users emails for readability

### Link to the issue
https://github.com/rubyforgood/pet-rescue/issues/30

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@
 )
 
 @user_one = User.create!(
-  email: 'a@b.com',
+  email: 'adopter1@example.com',
```
Comment (Thrillberg) [inline db/seeds.rb:10]: Love these changes! Can you update the README, too? I believe it references emails like `a@b.com`, `b@c.com`, etc.

**Comment by edwinthinks** [comment]
Update the readme please with the new credentials.

File: db/seeds.rb [hunk 2]
```diff
@@ -193,18 +193,18 @@
 )
 
 # create 10 pets
-i = 1
-until i == 10 do
+
+10.times  do
```
Comment (Thrillberg) [inline db/seeds.rb:197]: This is really great! Good use of idiomatic Ruby.

**Review by Thrillberg — COMMENTED** [review]
Looks great, thank you @jadekstewart3 ! Just one request in review. :D

**Review by Thrillberg — APPROVED** [review]
